code/main/splash/src/main/java/com/nicholasrutherford/chal/main/splash/SplashViewModel.kt

```diff
@@ -22,7 +22,7 @@ class SplashViewModel @ViewModelInject constructor(
             if (!firebaseAuth.isLoggedIn) {
                 navigation.showlogin()
             } else {
-                navigation.showlogin()
+                navigation.showNewsFeed()
```

Comment:
we should be prompting user to got to the news feed screen, if their logged in. 

Not the login screen, very easy fix.